docs/working-with-node/query-json-rpc.md

```diff
@@ -58,9 +62,22 @@ The command should return the following output:
 Now that we've confirmed the account we set up as premined has the correct balance, we can transfer some ether:
 
 ````bash
-polygon-edge txpool add --nonce 0 --from 0x1010101010101010101010101010101010101010 --to 0x0000000000000000000000000000000000000010 --value 0x100
-````
+var Web3 = require("web3");
+
+const web3 = new Web3("<provider's websocket jsonrpc address>"); //example: ws://localhost:10002/ws
+web3.eth.accounts
+  .signTransaction(
+    {
+      to: "<recipient public address>",
+      value: web3.utils.toWei("<value in ETH>"),
+      gas: 21000,
```

Comment:
`gas` may not be required?

Reply:
If you do not set gas when sending signTransaction, you will get. That's why i hardcoded this value as example
`return new Error('"gas" is missing');`